service-base/src/main/java/org/eclipse/hono/service/plan/PrometheusSumQueryResultObject.java

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.service.plan;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Encapsulates the JSON response for the prometheus query found by the get operation of the Prometheus HTTP API.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class PrometheusSumQueryResultObject {
```

Comment:
I am not sure I understand why we need to bind the query result to a class. It looks as if the relevant data can be easily extracted from the result JSON and it doesn't look like the other data contained in the result is used anywhere, or is it?

Reply:
@sophokles73 The value from the query result is not quite simple to extract. That's why I bind the query result to a class. IMHO it improves the readability. As an alternative, I have changed the existing code to extract the value from the result JSON instead of binding. You can take a look once I pushed the changes.